### ISSUE TITLE
Docker-Compose files should only use publicly accessible images

### ISSUE THREAD

**Issue by lucasmoten** [greymatter-io/nifi-sdk#13]
The following image references are not available to users as they are private
- deciphernow/gm-data:1.1.1
- deciphernow/gm-jwt/security:1.1.1

Without those referenced, once the docker-compose is brought up, the endpoints for processors in the templates should be configured to use endpoints for GM Data that are available to a user.  Likewise, establishing SSL Context information. Some guidance on this is available in the template documentation.

**Comment by lucasmoten** [comment]
In leui of docker images, the Remote URL that would reference GM Data from the NiFi templates can point to a valid running GM Data instance.

**Comment by TheMover** [comment]
Theoretically someone using this repo could have no access to any running gm-data source(a potential customer wanting to prototype etc.) It might be beneficial to build a small gm-data mock service that someone could reference to play around with these processors without having the license to gm-data yet.

**Comment by lucasmoten** [comment]
> Theoretically someone using this repo could have no access to any running gm-data source(a potential customer wanting to prototype etc.) It might be beneficial to build a small gm-data mock service that someone could reference to play around with these processors without having the license to gm-data yet.

That's true of all the Grey Matter services.  But entertaining this, the critical endpoints referenced by the processors are

* /config
* /self
* /list
* /write
